mlflow/models/utils.py

```diff
@@ -589,3 +591,97 @@ def validate_schema(data: PyFuncInput, expected_schema: Schema) -> None:
         mlflow.models.validate_schema(input_data, model_signature.inputs)
     """
     _enforce_schema(data, expected_schema)
+
+
+@experimental
+def add_libraries_to_model(model_uri, run_id=None, registered_model_name=None):
+    """
+    Given a registered model_uri (models:/<model_name>/<model_version/stage/latest>), this utility
+    re-logs the model along with all the required model libraries back to the model registry.
+    The required model libraries are stored along with the model as model artifacts. In
+    addition, supporting files to the model (e.g. conda.yaml, requirements.txt) are modified
+    to use the added libraries.
+
+    The default working of this util creates a new model version under the same registered model
+    name, this behavior can be overridden by passing in a registered_model_name to the utility.
+
+    :param model_uri: A registered model uri in the model registry of the form
+                        models:/<model_name>/<model_version/stage/latest>
```

Comment:
Nittiest of nits: Can we align the param docstrings?

```suggestion
    :param model_uri: A registered model uri in the model registry of the form
                      models:/<model_name>/<model_version/stage/latest>
```